### ISSUE TITLE
[BUG?] Infections aren't nerfed, you liars.

### ISSUE THREAD

**Issue by HzDonut** [VOREStation/VOREStation#5624]
#### Brief description of the issue
Many moons ago and after many fights, we all agreed the current speed of infection gain and spread was absolutely ridiculous and were promised nerfs. Either that nerf never came, or it got reverted, because holy shit can you get infected to hell and back near instantly now.

#### What you expected to happen
Infections would spread at a reasonable rate

#### What actually happened
Get severely infected about one minute after getting hurt

UPDATE: More like 30 seconds, actually.

UPDATE UPDATE: I was actually less than that, I was right next to the shuttle and my toxins skyrocketed from infections by the time I got inside. Two rooms worth of movement, about? (Note: I was already taking toxins, but the rate of gain doubled from infection.)

#### Steps to reproduce
- Get wounded
- Don't immediately treat it
- Suffer

#### Code Revision
- release - 
0015a40b20070fa8cc03fd2c80ca7655174b50da
 
#### Anything else you may wish to add:
- Mainly seems to be an issue in any situation you're wearing a hardsuit or voidsuit.

**Comment by Wickedtemp** [comment]
It goes by germ level. If your germ level is instantly increased to a certain threshold, an infection can immediately form, but that would normally require either an admin messing with variables or you'd have to walk on an insanely dirty tile. Do you know how far the infections had progressed, and the cause of the initial toxin gain?

**Comment by HzDonut** [comment]
I don't know how far the infections progressed because I didn't manage to sneak a peek at my body scanner results, but judging by the rate I was gaining toxins I'm going to guess I had at least one severe infection. They claim the initial toxins were from my voidsuit being punctured, but after I healed and went back to the ship, my suit was only "a bit scorched" which has never led to that happening any of the other times I got damage of that level. From my perspective all I saw was myself get hit with some bullets from a hivebot, get some bicard to get me healed up from "not dying", then limping back to the ship where, once inside, I was scanned and told I was infected to shit and gaining toxins.
